hack/drone.jsonnet

```diff
@@ -335,7 +335,17 @@ local integration_trigger = {
   },
 };
 
+local integration_nightly_trigger = {
```

Comment:
unfortunately trigger expressions only support `and` conditions, so we can't make pipeline run both as promote target and cron

Reply:
Ahh that's right. :(